bundle/definition/validators.go

```diff
@@ -1,45 +1,54 @@
 package definition
 
 import (
+	"context"
 	"encoding/base64"
 	"fmt"
 
+	"github.com/qri-io/jsonpointer"
 	"github.com/qri-io/jsonschema"
 )
 
 // ContentEncoding represents a "custom" Schema property
 type ContentEncoding string
 
 // NewContentEncoding allocates a new ContentEncoding validator
-func NewContentEncoding() jsonschema.Validator {
+func NewContentEncoding() jsonschema.Keyword {
 	return new(ContentEncoding)
 }
 
-// Validate implements the Validator interface for ContentEncoding
-// which, as of writing, isn't included by default in the jsonschema library we consume
-func (c ContentEncoding) Validate(propPath string, data interface{}, errs *[]jsonschema.ValError) {
+func (c ContentEncoding) Validate(propPath string, data interface{}, errs *[]jsonschema.KeyError) {}
```

Comment:
Is this the standard approach then with the latest jsonschema lib version?  Have `Validate` be a no-op and place all logic in `ValidateKeyword`?

Reply:
Probably not! Sorry about that, I followed what they did in the tests to define a custom keyword and I didn't dig further. Looks like there may be more that can be done.

Reply:
Here was their example for making a custom keyword:

https://github.com/qri-io/jsonschema/blob/master/README.md#custom-keywords

They just implemented ValidateKeyword and left the other functions empty.

I looked at a few implementations of string keywords and most also leave those functions empty. 

https://github.com/qri-io/jsonschema/blob/dbc3af1d666cc034a9ac89f10fba0ad6d5cb6c8e/keywords_string.go#L17-L27

An example of a keyword that does implement Register and resolve is `anyOf` which allows you to nest a subschema inside the keyword. That doesn't apply here though.
 
I don't think that the ContentEncoding keyword needs to implement them but if you see something that we should be doing in those functions, let me know. The doc is a bit scarce so I'm piecing it together from usage.

Reply:
Ah, thank you for the references.  Indeed, it looks like what you've done for ContentEncoding should be the right approach.  👍